plugins/factory-plugin/src/projects.ts

```diff
@@ -48,6 +48,9 @@ export function updateOrCreateGitProject(
             project.source.parameters = {};
         }
         project.source.parameters['branch'] = projectGitRemoteBranch;
+        delete project.source.parameters['startPoint'];
```

Comment:
why removing these informations ?

Reply:
OK I understand, if you do some changes you are not pointing anymore to a specific git revision.

Reply:
Yes, that was my reasoning as well. Hope this is the only place that needs changed for that...